GeneralsMD/Code/Main/WinMain.cpp

```diff
@@ -61,6 +61,7 @@
 #include "GameClient/IMEManager.h"
 #include "Win32Device/GameClient/Win32Mouse.h"
 #include "Win32Device/Common/Win32GameEngine.h"
+#include "WWLib/trim.h"
```

Comment:
Typing error in commit title:

"[ZH] Removal of duplicit strtrim"

"duplicate"

Reply:
Ah, that's probably not an English world, sorry. :) However, I think, this has already been [addressed](https://github.com/TheSuperHackers/GeneralsGameCode/pull/655) by other PR already in meantime...